themes/src/main/resources/theme/base/login/login.ftl

```diff
@@ -35,18 +35,18 @@
                                 </label>
                             </div>
                         </#if>
-                        </div>
```

Comment:
This `</div>` should not be removed, it is the end of `<div id="kc-form-options">`, this div is for "Remember me".
The `<div>` of "remember me" is same leave with "Forgot Password".

Please help to check my attached picture:
<img width="1332" alt="screen shot 2018-09-22 at 10 31 18 am" src="https://user-images.githubusercontent.com/701009/45912412-67edc980-be53-11e8-98b1-ba3bb0d97536.png">

Thanks
June

Reply:
Thanks for you review and feedback, @junezhang! 

I did it this way because tags are structured [same way](https://github.com/keycloak/keycloak/blob/eda09d2b323ac4c612c6b746e52e1af4534b3b80/themes/src/main/resources/theme/base/login/register.ftl#L73-L83) in the `register.ftl` template. If you're sure that this should be done differently in this case, please confirm. I'll update PR according to your screenshot.

Reply:
@devoto13 Sorry for replying late as taking leave this week.
Yes, I confirm the code structure.  For the 'register.ftl', It should be the same with the code structure in Login. Thanks.

Reply:
@junezhang updated as request.
Note that this is not consistent with the other forms, but ends up with the same DOM tree as the existing markup with the unpaired tags. Should be a good intermediate solution.

There seems to be more inconsistencies between forms, so I would prefer to only fix the problem at hand here. I'll see if I can put up another PR to unify forms in the `login` theme.

Reply:
@devoto13 I found another thing is `id="kc-form-wrapper"` and `id="kc-social-providers"` should be the same level. Others are LGTM.